src/components/molecules/NavSearchBar/NavSearchList.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { Key } from "react";
+import { FixedSizeList } from "react-window";
+
+import "./NavSearchList.scss";
+
+import {
+  NavSearchResultProps,
+  NavSearchResult,
+} from "components/molecules/NavSearchBar/NavSearchResult";
+
+// @debt refactor this into .scss or find more ergonomic component with same desired functionality as FixedSizeList
+// values are used here due to quirks of the FixedSizeList component
+// NOTE: keep in sync with the .scss files mentioned in comments
+const propsFor = (length: number) => ({
+  itemCount: length,
+  itemSize: 48, // 2 * $search-result-padding--vertical + $search-result-image--size; @see navSearchBarConstants.scss
+  height:
+    0 === length
```

Comment:
Thing you're testing on the left. Value it's being tested against on the right. ("If length is 0", not "If 0 is length")
```suggestion
    length === 0
```